app/views/sign_in/index.html.erb

```diff
@@ -20,10 +20,12 @@
       <%= govuk_link_to "Sign in using Persona", "/personas", { class: "govuk-button" }  %>
     <% end %>
 
+    <% if Settings.features.enable_support_link? %>
```

Comment:
There are instructions in the README about how to use `FeatureService` to check feature flags, rather than directly access the `Settings` config yourself.

Check up a couple of lines in this file (line 17) for an example.